build.gradle

```diff
@@ -11,11 +11,11 @@ ext {
 buildscript {
     // Build Script
     ext.android_gradle_plugin_version = '4.2.2'
-    ext.kotlin_version = '1.6.21'
+    ext.kotlin_version = '1.8.20'
```

Comment:
This is quite the jump, hope it's not going to cause any trouble for merchants 😬

Reply:
hope not 🤞